### DISCUSSION TITLE
The recommended rocm with Mi250X

### DISCUSSION THREAD

**Discussion by csccva** [AdaptiveCpp/AdaptiveCpp#873] [Q&A] [answered]
Hello,

What is the recommended `rocm` version for installing hipisycl on amd plaftorm with MI250X? I want to do some tests on our cluster and in their they have the `5.0.2` version. I know there are some other versions of rocm available.

Cristian

**Comment by illuhad** [comment, marked as answer]
I don't currently have access to MI250X. If your system has 5.0.2 preinstalled, I would suggest to just try that one?

**Comment by csccva** [comment]
Thanks for your reply. 

Let's how it goes. I will build the space environment from scratch. Hopefully not many trials are needed. The rocm is taking lots of time to compile.
